docs/experience/configure-requirements.md

```diff
@@ -0,0 +1,34 @@
+title: Configure Requirements
+description: Guide on how to configure Sematext Experience
+
+Different websites will have different performance requirements. We recommend a time threshold of 2s for page loads and HTTP requests and this is also the default we provide. You can change these settings in the Configure Requirements page. Time thresholds are defined in milliseconds.
```

Comment:
Different websites will have different performance requirements. We recommend a time threshold of 2s for page loads and HTTP requests and this is also the default we provide. You can change these settings in the Configure Requirements page. Time thresholds are defined in milliseconds.
->
Different websites will have different performance requirements. We recommend a time threshold of 2 seconds for page loads and HTTP requests. This is also the default we provide. You can change the settings on the Configure Requirements page. Time thresholds are defined in milliseconds.